clab/config.go

```diff
@@ -98,8 +99,7 @@ type Node struct {
 	User      string
 	Cmd       string
 	Env       []string
-	Mounts    map[string]volume
-	Volumes   map[string]struct{}
+	Mounts    []string // Bind mounts strings (src:dest:options)
```

Comment:
btw why add an additional fields here, `Mounts` we can use the already existing `Binds` field. 
It's also called binds on the docker api.
Otherwise, `&container.HostConfig.Mounts` can be used with `MountPoint.Type = 	mount.TypeBind`
Haven't tried it, so not sure it works.

Reply:
modified this in e4d5dd9
PTAL